game/src/plugins/util/command/src/command.kt

```diff
@@ -0,0 +1,19 @@
+import org.apollo.game.model.entity.Player
+
+/**
+ * Checks whether the amount of arguments provided is correct, sending the player the specified
+ * message if not.
+ */
+fun valid_arg_length(args: Array<String>, length: Any, player: Player, message: String): Boolean {

```

Comment:
I think its probably nicer to overload this method, providing one with `length: Int` and another with `length: IntRange`, to retain type safety